nsxt/policy_common.go

```diff
@@ -409,6 +409,27 @@ func setPolicyRulesInSchema(d *schema.ResourceData, rules []model.Rule) error {
 	return d.Set("rule", rulesList)
 }
 
+func validatePolicyRuleSequence(d *schema.ResourceData) error {
+	rules := d.Get("rule").([]interface{})
+	latestNum := int64(0)
+	for _, rule := range rules {
+		data := rule.(map[string]interface{})
+		seq := data["sequence_number"].(int)
+		sequenceNumber := int64(seq)
```

Comment:
Could be
`sequenceNumber := int64(data["sequence_number"].(int))`

as `seq` isn't used elsewhere